src/routes/customers/index.svelte

```diff
@@ -1 +1,121 @@
-customers
\ No newline at end of file
+<script lang="ts">
+	import Customers from '$lib/customers/Customers.svelte';
+	import { mediumAndDown, smallAndDown } from '$lib/stores/breakpoints';
+	import { onMount } from 'svelte';
+	import { Button, Card, CardActions } from 'svelte-materialify';
+	import { CardText, Select } from 'svelte-materialify';
+	import graphqlClient from '$lib/helpers/graphql';
```

Comment:
Initialize the `urql/svelte` in this file after imports https://svelte.dev/repl/fdd5034d023146f49614d3b087515df5?version=3.37.0